pkg/graveler/ref/manager.go

```diff
@@ -37,15 +39,16 @@ type CommitNode struct {
 	generation     int
 }
 
-func NewPGRefManager(executor batch.Batcher, db db.Database, addressProvider ident.AddressProvider) *Manager {
-	return &Manager{
+func NewKVPGRefManager(executor batch.Batcher, kvStore kv.StoreMessage, db db.Database, addressProvider ident.AddressProvider) *KVManager {
```

Comment:
```suggestion
func NewKVRefManager(executor batch.Batcher, kvStore kv.StoreMessage, db db.Database, addressProvider ident.AddressProvider) *KVManager {
```
Not sure what does the PG stands for, but eitherways the function returns `ref.KVManager`

Reply:
done